### DISCUSSION TITLE
Include version number in .exe file properties?

### DISCUSSION THREAD

**Discussion by RicoLeNecro** [diasurgical/DevilutionX#5457] [Ideas]
Hello there,

I've noticed that, as of current release 1.4.1, the devilutionx.exe files (both windows-i386 and windows-x86_64) doesn't seem to include any version information, when using Windows' explorer and showing the "Details" tab of said file.

I think being able to view those pieces of information that way could be useful, if possible including whether said .exe file is the 32 or 64 bits variant.

**Comment by AJenbo** [comment]
useful links
https://github.com/malaterre/PublicRep/tree/master/CMake/test3
https://github.com/diasurgical/devilutionX/blob/master/Packaging/windows/devilutionx.rc
https://github.com/halex2005/CMakeHelpers/blob/master/generate_product_version.cmake
